articles/container-apps/azure-arc-create-container-app.md

```diff
@@ -112,7 +113,7 @@ let StartTime = ago(72h);
 let EndTime = now();
 ContainerAppsConsoleLogs_CL
 | where TimeGenerated between (StartTime .. EndTime)
-| where AppName_s =~ "myContainerApp"
+| where AppName_s =~ "my-container-app"
```

Comment:
this cannot use $myContainerApp, since it is not a bash command, but in the log analysis portal.